src/Directory.Build.props

```diff
@@ -27,6 +27,8 @@
 		<DebugType>portable</DebugType>
 		<DebugSymbols>True</DebugSymbols>
 		<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+
+		<UnoExtensionsGeneration_DisableModuleInitializerAttribute>True</UnoExtensionsGeneration_DisableModuleInitializerAttribute>
```

Comment:
No we need it is lot of package, we must not set this in `Directory.Buidls.props`